### PR TITLE
Upgrade onflow/crypto

### PR DESCRIPTION
**Pull request by tarakby**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -49,7 +49,7 @@ require (
 	github.com/multiformats/go-multihash v0.2.3
 	github.com/onflow/atree v0.8.0-rc.5
 	github.com/onflow/cadence v1.0.0-preview.48
-	github.com/onflow/crypto v0.25.1
+	github.com/onflow/crypto v0.25.2-0.20240813163121-a1d4a1128629
```
Comment (peterargue) [inline go.mod:52]: can we use a tagged version?

Comment (tarakby) [reply go.mod:52]: you're right, I could tag a new crypto version

Comment (tarakby) [reply go.mod:52]: Before I tag a new version, I'd like to also add more dependency upgrades to the crypto module: https://github.com/onflow/crypto/pull/13

Comment (tarakby) [reply go.mod]: Updated to a tagged version https://github.com/onflow/flow-go/pull/6333/commits/819c2104dfd17c386d2184e836443b71296c270c

**Review by peterargue — APPROVED** [review]

**Pull request merged**